Sources/Workspace/Workspace.swift

```diff
@@ -1268,7 +1272,8 @@ extension Workspace {
                     _ = try clone(specifier: specifier, requirement: requirement)
                 case .removed:
                     try remove(specifier: specifier)
-                case .unchanged: break
+                case .unchanged: 
+                    delegate.repositoryRemainedUnchanged(specifier.url)
```

Comment:
I don't think its a good idea to print this for every repository. We can just show one message in case nothing was updated ([see](https://github.com/apple/swift-package-manager/pull/1338#issuecomment-329379013)).